apps/mull-ui/src/app/components/event-card/event-card.tsx

```diff
@@ -34,25 +36,29 @@ export const EventCard = ({ event, style = {}, onClick, isJoined = false }: Even
         <div>{time.replace(/\s/g, '')}</div>
       </div>
       {/* TODO: Add/remove user to event on press */}
```

Comment:
💡 This todo seems outdated, we should probably remove it.

Reply:
removed